sys/cbor/cbor.c

```diff
@@ -236,12 +233,12 @@ void dump_memory(const unsigned char *data, size_t size)
         return;
     }
 
-    DEBUG("0x");
+    printf("0x");
```

Comment:
Arghs sorry. I did not know exactly what `dump_memory()` does (though the name should have given me a hint ^^"). Here the `DEBUG()` macro was used wrong. For `dump_memory()` the `printf()` is okay. Not sure we need a pseudo-module for that, because usually link-time optimization removes functions that are not used (if one doesn't want to dump streams to stdout)